README.md

```diff
@@ -2,69 +2,147 @@
+
+<br>
+
+
+> ### **Official documentation**  
+> This README provides basic examples of how to get started with `react-native-calendars`. For detailed information, refer to the [official documentation site](https://wix.github.io/react-native-calendars/docs/intro).
 
-The package is both **Android** and **iOS** compatible.
 
-**See our new [Docs site](https://wix.github.io/react-native-calendars/)**
+## Features ✨
+* Pure JS. No Native code required
+* Customization of style, content (days, months, etc) and dates
+* Detailed documentation and examples
+* Swipeable calendar with flexible custom rendering
+* Scrolling to today, selecting dates, and more
+* Allowing or blocking certain dates
+* Accessibility support
+* Automatic date formatting for different locales
 
-## Try it out
 
-You can run example module by performing these steps:
+## Try it out ⚡
+
+You can run a sample module using these steps:
 
 ```
 $ git clone git@github.com:wix/react-native-calendars.git
+
 $ cd react-native-calendars
+
 $ npm install
+
 $ cd ios && pod install && cd ..
+
 $ react-native run-ios
 ```
 
 You can check example screens source code in [example module screens](https://github.com/wix-private/wix-react-native-calendar/tree/master/example/src/screens)
 
 This project is compatible with Expo/CRNA (without ejecting), and the examples have been [published on Expo](https://expo.io/@community/react-native-calendars-example)
 
-## Installation
 
-Using NPM:
+## Getting Started 🔧
+
+Here's how to get started with react-native-calendars in your React Native project:
+
+### Install the package:
+
+Using `npm`:
 ```
 $ npm install --save react-native-calendars
 ```
 
-Using Yarn:
+Using `Yarn`:
 ```
 $ yarn add react-native-calendars
 ```
 
-The solution is implemented in JavaScript so no native module linking is required.
+**RN Calendars is implemented in JavaScript, so no native module linking is required.**
+
+## Usage 🚀
+Basic usage examples of the library
+
+##### **For detailed information on using this component, see the [official documentation site](https://wix.github.io/react-native-calendars/docs/intro)**
+### Importing the `Calendar` component
+
+```javascript
+import {`[Calendar](#calendar), [CalendarList](#calendarlist), [Agenda](#agenda)`} from 'react-native-calendars';
+```
+### Use the `Calendar` component in your app:
+
+```javascript
+<Calendar
+  onDayPress={(day) => {console.log('selected day', day)}}
+/>
+```
 
-## Usage
 
-`import {`[Calendar](#calendar), [CalendarList](#calendarlist), [Agenda](#agenda)`} from 'react-native-calendars';`
+## Some Code Examples
 
-All parameters for components are optional. By default the month of current local date will be displayed.
+Here are a few code snippets that demonstrate how to use some of the key features of react-native-calendars:  
 
-Event handler callbacks are called with `calendar objects` like this:
 
+### Customize the appearance of the calendar:
```

Comment:
Feels like this should come after "Creating a basic calendar with default settings", no?

Reply:
Yes, definitely. It's the logical order for sure. Thanks!